Adafruit_MQTT.cpp

```diff
@@ -665,11 +666,22 @@ uint8_t Adafruit_MQTT::connectPacket(uint8_t *packet) {
   return len;
 }
 
+uint16_t Adafruit_MQTT::packetAdditionalLen(uint16_t currLen) {
```

Comment:
could this instead be incorporated into `publishPacket`?

Reply:
Okay. I will use that function in cpp file as a static. So this new line is now removed.